homeassistant/components/paperless_ngx/sensor.py

```diff
@@ -0,0 +1,109 @@
+"""Update platform for Paperless-ngx."""
+
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+
+from pypaperless.models import Statistic
+
+from homeassistant.components.sensor import (
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+    StateType,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddConfigEntryEntitiesCallback
+
+from .coordinator import PaperlessConfigEntry, PaperlessCoordinator
+from .entity import PaperlessEntity
+
+PARALLEL_UPDATES = 0
+
+
+@dataclass(frozen=True, kw_only=True)
+class PaperlessEntityDescription(SensorEntityDescription):
+    """Describes Paperless-ngx sensor entity."""
+
+    value_fn: Callable[[Statistic], int | None]
```

Comment:
```suggestion
    value_fn: Callable[[Statistic], int]
```
currently it never can be None can it?

Reply:
In the library it is marked as None, so I have to do the same here, the linter would trigger